Allow for setting of custom plus/minus indicators for Accordion.

jellisii opened the pull request:
Addresses #10413

jellisii committed:
Allow for setting of custom plus/minus indicators.

ncoden commented on scss/components/_accordion.scss:
```diff
@@ -14,6 +14,14 @@ $accordion-background: $white !default;
 
+/// Default indicator for plus when $accordion-plusminus is true
+/// @type String
+$accordion-plus-character: '\002B' !default;
```
Could you rename it to `$accordion-plus-content` as this is a String that can contain several characters ?

ncoden commented on scss/components/_accordion.scss:
```diff
@@ -14,6 +14,14 @@ $accordion-background: $white !default;
+
+/// Default indicator for minus when $accordion-plusminus is true
+/// @type String
+$accordion-minus-character: '\2013' !default;
```
`$accordion-minus-content`

ncoden commented on scss/components/_accordion.scss:
```diff
@@ -14,6 +14,14 @@ $accordion-background: $white !default;
 /// @type Boolean
 $accordion-plusminus: true !default;
 
+/// Default indicator for plus when $accordion-plusminus is true
```
You can add "`" around variable names and values like elsewhere in the docs

ncoden reviewed (changes requested):
👍 Thanks @jellisii 

It make me think that we should have these options for every component where it could be useful to customize them, for lrt/rtl support and design customization (with a custom webfont for example).

ncoden committed:
feat: add Pagination option to customize arrow icons

Similar to https://github.com/zurb/foundation-sites/pull/11025

> It make me think that we should have these options for every component where it could be useful to customize them, for lrt/rtl support and design customization (with a custom webfont for example).

ncoden committed:
style: rename Accordion icon content settings

See https://github.com/zurb/foundation-sites/pull/11025/files#r173285018

ncoden committed:
docs: improve Accordion icon content settings documentation

See https://github.com/zurb/foundation-sites/pull/11025/files#r173287607

ncoden reviewed (approved):
@jellisii I took care of the requested changes. Seems good to merge for `v6.5.0`

ncoden committed:
Merge branch 'develop' into jellisii/accordionPlusMinus

ncoden commented:
I added the same options for Pagination. I did not found others content where customization would be useful.
